### PR TITLE
Feature/api metadata i18n. Closes #779

### LINKED ISSUE #779: Add i18n tags to API Backend Metadata template
The Metadata template, in the View API Backend page, does not have i18n tags around all texts. Add i18n tags to remaining texts.

### PR DESCRIPTION
**Pull request by brylie**
Added i18n strings to view metadata page. This does not include AutoForm label translations, as that currently seems non-trivial.

### CODE REVIEW

**Comment by brylie** [comment]
@elnzv will you please review thie PR?

File: lib/i18n/en.i18n.json [hunk 1]
```diff
@@ -98,6 +98,14 @@
   "view_ApiBackend_ExportYaml": "Export .yaml",
   "view_ApiBackend_Metadata_Title": "Metadata",
   "view_ApiBackend_Metadata_NotFound": "No metadata found.",
+  "view_ApiBackend_Metadata_Organization_title": "Organization",
```
Comment (elnzv) [inline lib/i18n/en.i18n.json]: Add those strings to Finnish translation file as well.

**Comment by brylie** [comment]
It may not be necessary to add the translation strings to the fi.i18n.json. Did you read somewhere that this is a requirement?

**Comment by elnzv** [comment]
@brylie, no, that is not a requirement, but just more comfortable to translate, no need to search for new lines in en.json. 

Anyway, I added those myself, resolved conflicts.
Merging.

**Pull request merged**